.gitignore

```diff
@@ -21,6 +21,7 @@ Session.vim
 .project
 .favorites.json
 .settings/
+.vs/
```

Comment:
Could you avoid changes to this file?

Reply:
this is the visual studio ide creating the file. like .vscode.

with the extension i have just released [(rust-analy](https://marketplace.visualstudio.com/items?itemName=kitamstudios.RustAnalyzer) - for which this PR is, i do expect windows devs to use visual studio more often.

is it okay to retain it?